Preserve user data during update (#912)
Dasharo/docs: docs/guides/capsule-update.md

* _Firmware capsule_<br>
  The firmware update capsule file itself.  Should have `.cap` file extension.

> ## List of items preserved after performing an update

Reviewer: Maybe something like "What's preserved by an update", because the reader can see it's the list.

Author: https://github.com/Dasharo/docs/pull/912/commits/80dcc0ef34936f01a8f08d51fbba747f9f6656d6

| Type         | Notes                                  |
| ---          | ---                                    |

> | SMMSTORE     | holds UEFI Variables                   |

Author: something like that? let me know if it needs any more changes: https://github.com/Dasharo/docs/pull/912/commits/0f1647609252bff0faac03a35b0b5a7b64589018

| ROMHOLE      | only on MSI                            |
| SMBIOS       | unique data like serial number or UUID |

> | boot logo    | set by the user                        |

Author: ok updated: https://github.com/Dasharo/docs/pull/912/commits/f7e9d05edad6a62159f565222ff5cabdf4b61bc4

Preservation is done as a best effort. However some
 firmware changes are expected (e.g., current custom

>  logo is too large for the new firmware), thus a

Reviewer: ```suggestion
 logo can be too large for the new firmware), thus a
```

    Don't reboot or power off the device until the process is completed!

After either a successful or failed update, the machine should reboot

>  automatically. After that if everything succeeded you should have an

Reviewer: ```suggestion
 automatically. After that, if everything succeeded, you should have an
```

Author: https://github.com/Dasharo/docs/pull/912/commits/80dcc0ef34936f01a8f08d51fbba747f9f6656d6

>  updated firmware with migrated data from the previous version.

Reviewer: I think the words should be swapped:
```suggestion
 updated firmware with data migrated from the previous version.
```

Author: https://github.com/Dasharo/docs/pull/912/commits/80dcc0ef34936f01a8f08d51fbba747f9f6656d6
